src/AVFoundation/AVEdgeWidths.cs

```diff
@@ -75,8 +75,7 @@ public override string ToString ()
 
 		public override int GetHashCode ()
 		{
-			return (int) Left ^
-				(int) Top ^ (int) Right ^ (int) Bottom;
+			return HashCode.Combine ((int) Left, (int) Top, (int) Right, (int) Bottom);
```

Comment:
I think you can remove the casts now:

```suggestion
			return HashCode.Combine (Left, Top, Right, Bottom);
```